libr/include/r_core.h

```diff
@@ -564,6 +564,7 @@ R_API int r_core_set_file_by_name(RBin * bin, const char *name);
 R_API void r_core_debug_rr(RCore *core, RReg *reg, int mode);
 
 /* fortune */
+R_API RList *r_core_fortune_types(void);
 R_API void r_core_fortune_list_types(void);
```

Comment:
uhm this function should return rlist, imho, but adding a function or changing a function signature are abi breaking changes. you can make the function `R_IPI` (internal api) instead and mark it with `R2_580` as a comment or so, so we can change that api when 5.8 is out